sdk/search/Azure.Search.Documents/src/Azure.Search.Documents.csproj

```diff
@@ -13,9 +13,12 @@
     <PackageTags>Azure Cognitive Search;Azure Search Documents;Azure Search;Search;Cognitive;Search Engine;Azure</PackageTags>
     <DisableEnhancedAnalysis>true</DisableEnhancedAnalysis>
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
-    <!-- These are still firing but not valid -->
-    <NoWarn>$(NoWarn);AZC0004</NoWarn>
     <IncludeOperationsSharedSource>true</IncludeOperationsSharedSource>
+    <NoWarn>
+     $(NoWarn);
+     AZC0004;
+     AZC0035; <!-- Missing model factory methods.  #51224-->
```

Comment:
Can we remove this suppression and add what we need to the model factory as part of this work?